puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman_datacenter.yaml

```diff
@@ -0,0 +1,6 @@
+- project:
+    name: foreman_datacenter
+    repo: foreman_datacenter
```

Comment:
I don't see this project on https://github.com/theforeman?utf8=%E2%9C%93&q=foreman_datacenter&type=&language= and I think we can currently only build from theforeman github org.